fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -557,7 +550,7 @@ Reference<IDatabase> DLApi::createDatabase609(const char* clusterFilePath) {
 
 	auto clusterFuture = toThreadFuture<FdbCApi::FDBCluster*>(api, f, [](FdbCApi::FDBFuture* f, FdbCApi* api) {
 		FdbCApi::FDBCluster* cluster;
-		api->futureGetCluster(f, &cluster);
+		throwIfError(api->futureGetCluster(f, &cluster));
```

Comment:
The `ThreadFuture` handling for external C functions that return futures should already be propagating errors through the converted thread future, so it shouldn't be the case that you can get an error here. Most other usages write this code as `fdb_error_t e = ...; ASSERT(!e);`

Reply:
Done